server/service/apple_mdm.go

```diff
@@ -1464,11 +1465,7 @@ func (svc *Service) BatchSetMDMAppleProfiles(ctx context.Context, tmID *uint, tm
 		return ctxerr.Wrap(ctx, fleet.NewInvalidArgumentError("team_name", "cannot specify both team_id and team_name"))
 	}
 	if tmID != nil || tmName != nil {
-		license, err := svc.License(ctx)
-		if err != nil {
-			svc.authz.SkipAuthorization(ctx) // so that the error message is not replaced by "forbidden"
-			return err
-		}
+		license, _ := license.FromContext(ctx)
```

Comment:
The idea here is that it cannot be missing/nil because it is set in `fleet serve`, right?

Reply:
Correct. And all our uses of `license.FromContext(ctx)` are exactly like this.
Reason of this change is inability for GitOps to read AppConfig, and read AppConfig is checked in `svc.License`.